beacon-chain/core/time/slot_epoch.go

```diff
@@ -53,6 +53,11 @@ func HigherEqualThanAltairVersionAndEpoch(s state.BeaconState, e primitives.Epoc
 	return s.Version() >= version.Altair && e >= params.BeaconConfig().AltairForkEpoch
 }
 
+// PeerDASIsActive checks whether peerDAS is active at the provided slot.
+func PeerDASIsActive(slot primitives.Slot) bool {
```

Comment:
`PeerDASIsActive` ==> `isPeerDASActive`?

(This convention seems to be used in other places in the code and indicates clearly the function returns a bool.)

Reply:
I like the current version more because if you write `if PeerDasIsActive()` then it reads very smoothly. But we don't have a rule for this

Reply:
I was following from terms like `DenebEnabled` as a guide rather than posing it as a question. I don't feel too strongly but if we do start using `isPeerDASActive` I would prefer it to be standardized for all similar functions.